src/app/chip_data_model.gni

```diff
@@ -220,6 +220,13 @@ template("chip_data_model") {
           "${_app_root}/clusters/${cluster}/${cluster}.cpp",
           "${_app_root}/clusters/${cluster}/supported-modes-manager.h",
         ]
+      } else if (cluster == "mode-base-server") {
+        sources += [
+          "${_app_root}/clusters/mode-base-server/mode-base-cluster-objects.cpp",
```

Comment:
It seems like this sort of expression is not used in this file. I'll leave it as is. I also think it might make it easier for someone to search for where these file are included.

Reply:
> It seems like this sort of expression is not used in this file

It totally is @hicklin.  You can see it in the context of the diff quoted above!

I don't feel too strongly about this, though.